README.md

```diff
@@ -21,6 +21,7 @@ You can find source files under the [`src`](https://github.com/IgniteUI/igniteui
 
 |Components|Status|||Directives|Status|||
 |:--|:--:|:--|:--|:--|:--:|:--|:--|
+|accordion|:white_check_mark:|[Readme](https://github.com/IgniteUI/igniteui-angular/blob/master/projects/igniteui-angular/src/lib/accordion/README.md)|[Docs](https://www.infragistics.com/products/ignite-ui-angular/angular/components/accordion)|||||

```

Comment:
```suggestion
|accordion|:white_check_mark:|[Readme](https://github.com/IgniteUI/igniteui-angular/blob/master/projects/igniteui-angular/src/lib/accordion/README.md)|[Docs](https://www.infragistics.com/products/ignite-ui-angular/angular/components/accordion)|
```
Remove the extra columns at the end, as it [adds an empty line](https://github.com/IgniteUI/igniteui-angular/tree/mevtimov/feat-9559-master) under `Directives` section